modules/ossm-config-smcp-kiali.adoc

```diff
@@ -0,0 +1,206 @@
+
+
+
+.Example SMCP parameters for Kiali
+[source,yaml]
+----
+apiVersion: maistra.io/v2
+kind: ServiceMeshControlPlane
+metadata:
+  name: basic
+spec:
+  addons:
+    kiali:
+      name: kiali
+      enabled: true
+      install:
+        dashboard:
+          viewOnly: false
+          enableGrafana: true
+          enableTracing: true
+          enablePrometheus: true
+        service:
+          ingress:
+            contextPath: /kiali
+----
+
+.Kiali parameters in the SMCP
+[options="header"]
+[cols="l, a, a, a"]
+|===
+|Parameter |Description |Values |Default value
+|spec:
+  addons:
+    kiali:
+      name:
+|Name of Kiali custom resource. If a Kiali CR matching the value of `name` exists, the {ProductShortname} Operator will use that CR for the installation. If no Kiali CR exists, the Operator will create one using this `name` and the configuration options specified in the SMCP.
+|string
+|`kiali`
+
+|kiali:
+  enabled:
+|This parameter enables/disables Kiali. Kiali is enabled by default.
+|`true`/`false`
+|`true`
+
+|kiali:
+  install:
+|Install a Kiali resource if the named Kiali resource is not present.
+|
+|
+
+|kiali:
+  install:
+    dashboard:
+|Configuration parameters for the dashboards shipped with Kiali.
+|
+|
+
+|kiali:
+  install:
+    dashboard:
+      viewOnly:
+|This parameter enables/disables view-only mode for the Kiali console. When view-only mode is enabled, users cannot use the Kiali console to make changes to the {ProductShortname}.
+|`true`/`false`
+|`false`
+
+|kiali:
+  install:
+    dashboard:
+      enableGrafana:
+|Grafana endpoint configured based on `spec.addons.grafana` configuration.
+|`true`/`false`
+|`true`
+
+|kiali:
+  install:
+    dashboard:
+      enablePrometheus:
+|Prometheus endpoint configured based on `spec.addons.prometheus` configuration.
+|`true`/`false`
+|`true`
+
+|kiali:
+  install:
+    dashboard:
+      enableTracing:
+|Tracing endpoint configured based on Jaeger custom resource configuration.
+|`true`/`false`
+|`true`
+
+|kiali:
+  install:
+    service:
+|Configuration parameters for the Kubernetes service associated with the Kiali installation.
+|
+|
+
+|kiali:
+  install:
+    service:
+      metadata:
+|Use to specify additional metadata to apply to resources.
+|N/A
+|N/A
+
+|kiali:
+  install:
+    service:
+      metadata:
+        annotations:
+|Use to specify additional annotations to be applied to the component's service.
+|string
+|N/A
+
+|kiali:
+  install:
+    service:
+      metadata:
+        labels:
+|Use to specify additional labels to be applied to the component's service.
+|string
+|N/A
+
+|kiali:
+  install:
+    service:
+      ingress:
+|Used to specify details for accessing the component’s service through a Kubernetes Ingress or an OpenShift Route.
```

Comment:
Because these docs are within the context of OpenShift, remove the "a Kubernetes Ingress or" phrase. When installing on OpenShift, it will install a Route, never a Kubernetes Ingress.
```suggestion
|Used to specify details for accessing the component’s service through an OpenShift Route.
```